api/Document.json

```diff
@@ -9282,4 +9282,4 @@
       }
     }
   }
-}
+}
```

Comment:
What's this for?

Reply:
I have no clue, reverting this little change.  :P